apps/experiments/migrations/0043_update_AWS_synthetic_voices.py

```diff
@@ -1,37 +1,15 @@
 # Generated by Django 4.2 on 2023-10-11 15:18
-import os, json
+import os
 from django.db import migrations
+from apps.utils import migration_utils
 
 
 def load_language_code(apps, schema_editor):
     SyntheticVoice = apps.get_model("experiments", "SyntheticVoice")
-    voice_data = {}
     current_directory = os.path.dirname(os.path.abspath(__file__))
     file_path = os.path.join(current_directory, "preload_data/aws_voices.json")
+    migration_utils.create_synthetic_voices_from_file(SyntheticVoice, file_path=file_path)
```

Comment:
I forgot to use the utility here. Context: We created AWS voices, but then later wanted to add `language_code` and `service` to the model as well. We then reran the migration to load the voices, only this time with `SyntheticVoice.objects.update_or_create(...)`, which is why I update the util function to use `update_or_create` rather than `get_or_create`. The json file is the latest one (so it has the concept of "service" and "language code", so no need to accomodate versions of this file)

Reply:
Oh sorry I forgot to mention, I realized that we can't do that. Reason being that we added the `SyntheticVoice` in [migration 0023](https://github.com/dimagi/open-chat-studio/blob/main/apps/experiments/migrations/0023_syntheticvoice_experiment_synthetic_voice.py). We loaded the AWS voices in [migration 32](https://github.com/dimagi/open-chat-studio/blob/main/apps/experiments/migrations/0032_load_synthetic_voices.py), and then altered the synthetic voice model in [migration 42](https://github.com/dimagi/open-chat-studio/blob/main/apps/experiments/migrations/0042_alter_syntheticvoice_unique_together_and_more.py), meaning that even though the first migration ran, it will ignore the `service` and `language_code` fields from the json file. In the second migration where we load the voices, all those records will be updated, so we sort of need this one to also run :+1: